lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1529,6 +1538,32 @@ OpFoldResult Aten__Contains__StrOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// Aten__Contains__IntListOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult Aten__Contains__IntListOp::fold(ArrayRef<Attribute> operands) {
+  auto arrayConstruct = isListConstructNotModified(l());
+  auto itemConstruct = item();
+  if (!arrayConstruct)
+    return nullptr;
```

Comment:
Once the return value of `isListConstructNotModified` is changed to `bool`, this should become

```cpp
if (!isListConstructNotModified(l()))
  return nullptr;
```